hack/generate-swagger-api.ps1

```diff
@@ -0,0 +1,24 @@
+swagger generate model -f api/swagger.yaml `
+    -t api -m types --skip-validator -C api/swagger-gen.yaml `
+    -n ErrorResponse `
+    -n GraphDriverData `
+    -n IdResponse `
+    -n ImageDeleteResponseItem `
+    -n ImageSummary `
+    -n Plugin -n PluginDevice -n PluginMount -n PluginEnv -n PluginInterfaceType `
+    -n Port `
+    -n ServiceUpdateResponse `
+    -n Volume
+
+swagger generate operation -f api/swagger.yaml `
+    -t api -a types -m types -C api/swagger-gen.yaml `
+    -T api/templates --skip-responses --skip-parameters --skip-validator `
+    -n Authenticate `
```

Comment:
Can we add a comment here, and in the Linux counterpart that when updating this list, the corresponding file for the other platform must be updated as well?

Reply:
@thaJeztah Ooops. Totally missed that comment. Done now.